### ISSUE TITLE
Package addition: Leocad

### ISSUE THREAD

**Issue by nathaneltitane** [termux/x11-packages#216]
**Package description**

**Link to home page and sources**
1. Home page: https://www.leocad.org
2. Source code: https://github.com/leozide/leocad

**Additional information**
Have tested on LoD using Samasung Galaxy s10+ and confirm the package can be built and runs using chroot systems.

Only issue is that the application requires a qt environment override:

QT_X11_NO_MITSHM=1
QT_GRAPHICSSYSTEM=native

**Comment by nathaneltitane** [comment]
@xeffyr - what would be needed to get this binary to compile on termux runlevel rather than distro chroot?

**Comment by xeffyr** [comment]
A proper `qt5-base` package. Not a one currently present which has lots of issues. ETA of fixing `qt5-base` is unspecified.

**Comment by nathaneltitane** [comment]
> A proper `qt5-base` package. Not a one currently present which has lots of issues. ETA of fixing `qt5-base` is unspecified.

Thanks @xeffyr

**Comment by nathaneltitane** [comment]
@xeffyr I see that package qt5-qtbase has been added?

**Comment by Yisus7u7** [comment]
Very true

**Comment by xeffyr** [comment]
@nathaneltitane Yes, this one should be complete and also more Qt modules are available besides the basic.

**Comment by nathaneltitane** [comment]
> @nathaneltitane Yes, this one should be complete and also more Qt modules are available besides the basic.

Could the package be pulled and built?
I'll try to manually build on my side on termux directly and see what gives.

**Comment by Yisus7u7** [comment]
> > @nathaneltitane Yes, this one should be complete and also more Qt modules are available besides the basic.
> 
> Could the package be pulled and built?
> I'll try to manually build on my side on termux directly and see what gives.

X2 XD

**Comment by Yisus7u7** [comment]
@nathaneltitane

I do not think it will be possible to compile at the moment because of this : 

~/Documents/leocad $ qmake leocad.pro
Info: creating stash file /data/data/com.termux/files/home/Documents/leocad/.qmake.stash
Project ERROR: Unknown module(s) in QT: opengl

Termux lacks opengl :C

**Comment by nathaneltitane** [comment]
> @nathaneltitane
> 
> I do not think it will be possible to compile at the moment because of this :
> 
> ~/Documents/leocad $ qmake leocad.pro
> Info: creating stash file /data/data/com.termux/files/home/Documents/leocad/.qmake.stash
> Project ERROR: Unknown module(s) in QT: opengl
> 
> Termux lacks opengl :C

yikes...
@xeffyr I'm sure it could compile and run under llvm if opengl was present in repos
